app/uk/gov/hmrc/gform/views/html/package.scala

```diff
@@ -75,4 +78,18 @@ package object html {
       href = Some(s"/submissions/test-only/payloads/${formTemplate._id.value}/${accessCode.value}"),
       attributes = Map.empty
     )
+
+  private def startNewFormFooterItem(formTemplate: FormTemplate) =
+    new FooterItem(
+      text = Some("Start new form"),
+      href = Some(s"/submissions/new-form/${formTemplate._id.value}"),
+      attributes = Map.empty
+    )
+
+  private def startNewFormFooterItemWithAccessCode(formTemplate: FormTemplate, accessCode: AccessCode) =
+    new FooterItem(
+      text = Some("Start new form"),
+      href = Some(s"/submissions/new-form/${formTemplate._id.value}/${accessCode.value}"),
+      attributes = Map.empty
+    )
```

Comment:
What is this link? There is only one endpoint (unless I'm missing something) to start a new form and that is `/new-form/:formTemplateId`